app/Http/Frontend/Controllers/ProductsController.php

```diff
@@ -11,13 +11,49 @@ class ProductsController extends Controller
 {
 
     /**
+     * Display a listing of the new products.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
      * Show the application dashboard.
      *
      * @return \Illuminate\Http\Response
      */
     public function index()
     {
-        $products = Product::select('id', 'name', 'image', 'price')->orDerBy('id', 'DESC')->skip(0)->take(6)->get();
+        $products = Product::select('id', 'name', 'image', 'price', 'keywords')->orDerBy('id', 'DESC')->skip(0)->take(6)->get();
```

Comment:
this keywords apply for route of add products to cart, keywords == name product.